v7.0 Changelog (#5677)
mattermost/docs: source/upgrade/important-upgrade-notes.rst

| If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
+====================================================+==================================================================================================================================================================+
| v7.0                                               | MySQL self-hosted customers may notice the migration taking longer than usual when having a large number of rows in FileInfo table. For MySQL, it takes around   |
|                                                    | 19s for a table of size 0.7M rows. The time required for PostgreSQL is negligible.                                                                               |
|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+

> |                                                    | When a new config setting ``EnableAppBar`` is enabled, all channel header icons registered by plugins will be moved to the new App Bar, even if they do not      |

Reviewer: ```suggestion
|                                                    | When a new config setting ``EnableAppBar`` is enabled, all channel header icons registered by plugins will be moved to the new Apps Bar, even if they do not      |
```

> |                                                    | When a new config setting ``EnableAppBar`` is enabled, all channel header icons registered by plugins will be moved to the new Apps Bar, even if they do not     |

Reviewer 2: ```suggestion
|                                                    | When a new config setting ``ExperimentalSettings.EnableAppBar`` is enabled, all channel header icons registered by plugins will be moved to the new Apps Bar, even if they do not     |
```

|                                                    | explicitly use the new registry function to render a component there. The setting ``EnableAppBar`` defaults to ``false`` for self-hosted deployments.            |
|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
|                                                    | The value of ``ServiceSettings.TrustedProxyIPHeader`` defaults to empty from now on. A previous bug prevented this from happening in certain conditions.         |
|                                                    | Customers are requested to check for these values in their config and set them to nil if necessary.                                                              |
|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+

> |                                                    | **IMPORTANT:** Session length configuration settings have changed from days to hours. If anything goes awry during the server upgrade,                           |

Reviewer 2: Since this is an important note, I'd move it first on the list.

|                                                    | server upgrade, it could result in users not being able to log in. For customers who use environment variables for those config settings, the automatic migration|

> |                                                    | will not work and they'll need to add new environment variables:                                                                                                 |

Reviewer: ```suggestion
|                                                    | won't work, and new configuration environment variables must be added: for:                                                                                                 |
```

|                                                    | to ``false`` for self-hosted deployments.                                                                                                                        |
|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
|                                                    | The value of ``ServiceSettings.TrustedProxyIPHeader`` defaults to empty from now on. A previous bug prevented this from happening in certain conditions.         |
|                                                    | Customers are requested to check for these values in their config and set them to nil if necessary. See more details                                             |
|                                                    | `here <https://docs.mattermost.com/configure/configuration-settings.html#trusted-proxy-ip-header>`_.                                                             |            

> +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+

Reviewer 3: Collapsed Reply Threads (link to docs) is now generally available and enabled by default for new Mattermost servers. For servers upgrading to v7.0 and later, please reference [this article](link to KB article from @cwarnermm) for more information and guidance for enabling the feature.

Reviewer: The URL of the new KB article on CRT GA will be: https://mattermost.zendesk.com/knowledge/articles/6880701948564/en-us?brand_id=327479

Note that this KB article is currently in draft and not accessible from the KB site. It will be published live on June 14th.

|                                                    | Customers are requested to check for these values in their config and set them to nil if necessary. See more details                                             |
|                                                    | `here <https://docs.mattermost.com/configure/configuration-settings.html#trusted-proxy-ip-header>`_.                                                             |
|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
|                                                    | `Collapsed Reply Threads <https://docs.mattermost.com/channels/organize-conversations.html>`_ is now generally available and enabled by default for new          |
|                                                    | Mattermost servers. For servers upgrading to v7.0 and later, please reference                                                                                    |

> |                                                    | `this article <https://mattermost.zendesk.com/knowledge/articles/6880701948564/en-us?brand_id=327479>`_ for more information and guidance on enabling the        |

Reviewer: ```suggestion
|                                                    | `this article <https://support.mattermost.com/hc/en-us/articles/6880701948564>`_ for more information and guidance on enabling the        |
```

Reviewer: Note: this change likely breaks the table format
